Index: springboot-backend/src/main/java/com/balwinderjava/springboot/exception/ShouldNotHappenException.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/springboot-backend/src/main/java/com/balwinderjava/springboot/exception/ShouldNotHappenException.java b/springboot-backend/src/main/java/com/balwinderjava/springboot/exception/ShouldNotHappenException.java
new file mode 100644
--- /dev/null	(date 1749699387850)
+++ b/springboot-backend/src/main/java/com/balwinderjava/springboot/exception/ShouldNotHappenException.java	(date 1749699387850)
@@ -0,0 +1,18 @@
+package com.balwinderjava.springboot.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
+public class ShouldNotHappenException extends RuntimeException{
+
+	private static final long serialVersionUID = 1L;
+	
+	public ShouldNotHappenException(String message) {
+		super(message);
+	}
+
+	public ShouldNotHappenException(String message, Throwable cause) {
+		super(message, cause);
+	}
+}
Index: springboot-backend/src/main/java/com/balwinderjava/springboot/SpringbootBackendApplication.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/springboot-backend/src/main/java/com/balwinderjava/springboot/SpringbootBackendApplication.java b/springboot-backend/src/main/java/com/balwinderjava/springboot/SpringbootBackendApplication.java
new file mode 100644
--- /dev/null	(date 1749698245333)
+++ b/springboot-backend/src/main/java/com/balwinderjava/springboot/SpringbootBackendApplication.java	(date 1749698245333)
@@ -0,0 +1,13 @@
+package com.balwinderjava.springboot;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class SpringbootBackendApplication {
+
+	public static void main(String[] args) {
+		SpringApplication.run(SpringbootBackendApplication.class, args);
+	}
+
+}
